Data/models/DataController.swift

```diff
@@ -23,6 +23,8 @@ public class DataController {
     private static let databaseName = "Brave.sqlite"
     private static let modelName = "Model"
     
+    private static let storeExistsErrorCode = 134081
```

Comment:
Can we comment here what is the error code for and why we need it specifically ?

Reply:
the constant name is `what`, i can add in comment `why`